client/src/pages/Product.tsx

```diff
@@ -227,7 +232,7 @@ const Footer = styled.div`
   }
 `;
 
-const LikeButton = styled.button`
+const LikeButtonContainer = styled.button`
```

Comment:
콘솔에서 버튼 안에 버튼이 들어가면 안 된다는 워닝이 뜨네요. 컨테이너를 div로 바꾸는 게 좋을 것 같아요~

```suggestion
const LikeButtonContainer = styled.div`
```

Reply:
콘솔 확인해볼 생각을 못했네요 찾아주셔서 감사합니다